src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASS_LENGTH = 6;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+    private String login = "slavik";
+    private int age = 18;
+    private final String password = "Sukhov28";
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(age++);
+        user.setLogin(login);
+        user.setPassword(password);
+    }
+
+    @Test
+    void register_UserExistsAlready_Ok() {
```

Comment:
Is it really "ok"? (last part of method's name). Looks like we check "notOk" behavior in this test